test/sqllogictest/arithmetic.slt

```diff
@@ -498,6 +498,79 @@ SELECT round(CAST (5.0 AS float), 3.0)
 query error Cannot call function round\(double precision, double precision\): arguments cannot be implicitly cast to any implementation's parameters; try providing explicit casts
 SELECT round(CAST (5.0 AS float), CAST (3.0 AS float))
 
+query R
```

Comment:
Can you add tests for nan, inf, and -0?

Reply:
Done! Assuming this produces the desired output
```
query RRRR
SELECT trunc('NaN'::decimal), trunc('Inf'::float), trunc(0), trunc(-0)
----
NaN  inf  0  0
```
I confirmed that this is postgresql's behavior too.